crates/sui/src/unit_tests/snapshots/sui__upgrade_compatibility__upgrade_compatibility_tests__enum.snap

```diff
@@ -0,0 +1,210 @@
+  │
+  = Enums are part of a module's public interface and cannot be changed during an upgrade.
+  = Restore the original enum's abilities for enum 'EnumAddAbility' including the ordering.
+
+error[Compatibility E01003]: ability mismatch
+   ┌─ /Users/jordanjennings/code/sui/crates/sui/src/unit_tests/fixtures/upgrade_errors/enum_errors_v2/sources/UpgradeErrors.move:13:17
+   │
+13 │     public enum EnumRemoveAbility has copy { // drop removed
+   │                 ^^^^^^^^^^^^^^^^^ Missing abilities 'drop'
+   │
+   = Enums are part of a module's public interface and cannot be changed during an upgrade.
+   = Restore the original enum's abilities for enum 'EnumRemoveAbility' including the ordering.
+
+error[Compatibility E01003]: ability mismatch
+   ┌─ /Users/jordanjennings/code/sui/crates/sui/src/unit_tests/fixtures/upgrade_errors/enum_errors_v2/sources/UpgradeErrors.move:17:17
+   │
+17 │     public enum EnumAddAndRemoveAbility has copy, store {
+   │                 ^^^^^^^^^^^^^^^^^^^^^^^ Mismatched abilities: missing 'drop', unexpected 'store'
+   │
+   = Enums are part of a module's public interface and cannot be changed during an upgrade.
+   = Restore the original enum's abilities for enum 'EnumAddAndRemoveAbility' including the ordering.
+
+error[Compatibility E01002]: type mismatch
+   ┌─ /Users/jordanjennings/code/sui/crates/sui/src/unit_tests/fixtures/upgrade_errors/enum_errors_v2/sources/UpgradeErrors.move:23:9
+   │
+21 │     public enum EnumAddVariant {
+   │                 -------------- Enum definition
+22 │         A,
+23 │         B, // added
+   │         ^ New unexpected variant 'B'.
+   │
+   = Enums are part of a module's public interface and cannot be changed during an upgrade.
+   = Restore the original enum's variants for enum 'EnumAddVariant' including the ordering.
+
+error[Compatibility E01002]: type mismatch
+   ┌─ /Users/jordanjennings/code/sui/crates/sui/src/unit_tests/fixtures/upgrade_errors/enum_errors_v2/sources/UpgradeErrors.move:26:17
+   │
+26 │     public enum EnumRemoveVariant {
+   │                 ^^^^^^^^^^^^^^^^^ Missing variant 'B'.
+   │
+   = Enums are part of a module's public interface and cannot be changed during an upgrade.
+   = Restore the original enum's variant 'B' for enum 'EnumRemoveVariant' including the ordering.
+
+error[Compatibility E01002]: type mismatch
+   ┌─ /Users/jordanjennings/code/sui/crates/sui/src/unit_tests/fixtures/upgrade_errors/enum_errors_v2/sources/UpgradeErrors.move:33:9
+   │
+31 │     public enum EnumChangeVariant {
+   │                 ----------------- Enum definition
+32 │         A,
+33 │         C, // changed from B
+   │         ^ Mismatched variant name 'C', expected 'B'.
+   │
+   = Enums are part of a module's public interface and cannot be changed during an upgrade
+   = Restore the original enum's variants for enum 'EnumChangeVariant' including the ordering.
+
+error[Compatibility E01002]: type mismatch
```

Comment:
Is this really a "type mismatch"?

Reply:
The only other idea I had was layout mismatch?